Task13/Azure/index.html

```diff
@@ -5,21 +5,146 @@
     <title>Cloud Computing Services | Microsoft Azure</title>
     <link rel="icon" href="img/icon.svg">
     <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width,1 initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
 </head>
 
 <body>
-       <div class="banner">
-           <div class="azure">
-        <img src="img/menu.jpg" width="80px" class="menu">
-        <h2>Azure</h2>
-           </div>
-           <div class="icon">
-        <img src="img/icon.svg" class="icon" width="104px">
-           </div>
-           <div class="signin">
-        <img src="img/search.png" alt="search" width="104px">
-        <a href="">Sign In</a>
-           </div>
-       </div>
-    
-    
\ No newline at end of file
+    <div class="chat">
```

Comment:
Arrange the tags in their order
<html>
<head></head>
<body>
     <header></header>
     <section></section>
     .
     .
     .
     .
     <footer></footer>
</body>
</html

Reply:
Add header tag